x/concentrated-liquidity/swaps.go

```diff
@@ -403,6 +429,15 @@ func (k Keeper) computeOutAmtGivenIn(
 		return SwapResult{}, PoolUpdates{}, err
 	}
 
+	scalingFactor := oneDec
+	if updateAccumulators {
+		// We only need the scaling factor if we are updating the accumulators
```

Comment:
If this is the case, thoughts on making the above `var scalingFactor sdk.Dec` to guarantee it doesn't accidentally get used somewhere?

Reply:
Yeah that SGTM, done here https://github.com/osmosis-labs/osmosis/pull/7876/commits/60accca4ba22a7a0e1b0dac181051e4eebe5086e